Sources/Public/CalendarView.swift

```diff
@@ -707,6 +668,87 @@ public final class CalendarView: UIView {
+        size = CGSize(width: bounds.size.width, height: bounds.size.height * 3)
+
+      case .horizontal:
+        offset = CGPoint(
+          x: scrollView.contentOffset.x - bounds.width,
+          y: scrollView.contentOffset.y)
+        size = CGSize(width: bounds.size.width * 3, height: bounds.size.height)
+      }
+    } else {
+      offset = scrollView.contentOffset
+      size = bounds.size
+    }
+
+    let visibleItemsProvider = visibleItemsProvider(forSize: size)
+
+    scrollView.performWithoutNotifyingDelegate {
+      scrollMetricsMutator.setUpInitialMetricsIfNeeded()
+      scrollMetricsMutator.updateContentSizePerpendicularToScrollAxis(viewportSize: bounds.size)
+    }
+
+    let anchorLayoutItem: LayoutItem
+    if let scrollToItemContext = scrollToItemContext, !scrollToItemContext.animated {
+      anchorLayoutItem = self.anchorLayoutItem(
+        for: scrollToItemContext,
+        visibleItemsProvider: visibleItemsProvider)
+      // Clear the `scrollToItemContext` once we use it. This could happen over the course of
+      // several layout pass attempts since `isReadyForLayout` might be false initially.
+      self.scrollToItemContext = nil
+    } else if let previousAnchorLayoutItem = self.anchorLayoutItem {
+      anchorLayoutItem = previousAnchorLayoutItem
+    } else {
+      let initialScrollToItemContext = ScrollToItemContext(
+        targetItem: .month(content.monthRange.lowerBound),
+        scrollPosition: .firstFullyVisiblePosition,
+        animated: false)
+      anchorLayoutItem = self.anchorLayoutItem(
+        for: initialScrollToItemContext,
+        visibleItemsProvider: visibleItemsProvider)
+    }
+
+    let currentVisibleItemsDetails = visibleItemsProvider.detailsForVisibleItems(
+      surroundingPreviouslyVisibleLayoutItem: anchorLayoutItem,
+      offset: offset)
+    self.anchorLayoutItem = currentVisibleItemsDetails.centermostLayoutItem
+
+    updateVisibleViews(
+      withVisibleItems: currentVisibleItemsDetails.visibleItems,
+      previouslyVisibleItems: visibleItemsDetails?.visibleItems ?? [])
+
+    visibleItemsDetails = currentVisibleItemsDetails
+
+    let minimumScrollOffset = visibleItemsDetails?.contentStartBoundary.map {
+      ($0 - firstLayoutMarginValue).alignedToPixel(forScreenWithScale: scale)
+    }
+    let maximumScrollOffset = visibleItemsDetails?.contentEndBoundary.map {
+      ($0 + lastLayoutMarginValue).alignedToPixel(forScreenWithScale: scale)
+    }
+    scrollView.performWithoutNotifyingDelegate {
+      scrollMetricsMutator.updateScrollBoundaries(
+        minimumScrollOffset: minimumScrollOffset,
+        maximumScrollOffset: maximumScrollOffset)
+    }
+
+    cachedAccessibilityElements = nil
+    if let element = focusedAccessibilityElement as? OffScreenCalendarItemAccessibilityElement {
+      UIAccessibility.post(
+        notification: .screenChanged,
+        argument: visibleViewsForVisibleItems[element.correspondingItem])
+    }
+  }
+
```

Comment:
this is just a copy-paste from the original `layoutSubviews` implementation. No logic change in these lines.